src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -142,13 +142,28 @@ public void testInsertWhenValidData() throws SQLException {
         assertTrue(actual);
     }
 
+    @Test
+    public void testDeleteWhenValidData() throws SQLException {
+        createTableWithData(TABLE_NAME);
+        databaseManager.connect(dbName, USER, PASSWORD);
+        boolean actual = databaseManager.delete(TABLE_NAME, 1);
+        executeQuery(String.format("DROP TABLE IF EXISTS %s", TABLE_NAME));
+        assertTrue(actual);
+    }
+
     @Test
     public void testInsertWhenTableNotExists() {
         DataSet dataSet = createDataSet(new String[]{"1", "name1", "25"});
         databaseManager.connect(dbName, USER, PASSWORD);
         assertFalse(databaseManager.insert("noTable", dataSet));
     }
 
+    @Test
+    public void testDeleteWhenTableNotExists() {
```

Comment:
You provide testDeleteWhenValidData, testDeleteWhenTableNotExists and what about invalid data like not existed or id?

Reply:
I Agree with @Shtramak

Reply:
It seems to me that testing the case when id does not exist for my code is not necessary, since the result must come true in any case, whether the id is or not. But considering that the more tests, the better, maybe I'll add this case to the next PR. 
Thank you for your comments.

Reply:
I've missed that point... Then,  probably, you haven't provided all of the requirements of the task. My method works in other way - it returns false in case when id doesn't exist. Or if your implementation fits the task requirements then mine method works wrong. Looks like a point for discussion)

Reply:
Taking into considerstion the results of conversation in slack, in case of not existed id method delete should return false and this case sould be tested